### ISSUE TITLE
Add reference to `embed` object on channel->createMessage

### ISSUE THREAD

**Issue by Arteneko** [restcord/restcord#66]
I'm looking at the [Create Message](http://www.restcord.com/Channel/Create%20Message/) API doc, and it doesn't details the structure of `embed`.

It'd be nice to either have the structure inside the doc, or have a link to the discord API that details such structure.

For ref., the `embed` object structure documentation can be found [here](https://discordapp.com/developers/docs/resources/channel#embed-object).

**Comment by LeigerGaming** [comment]
I agree that the Restcord documentation is quite basic. I find myself just referring to the official Discord API docs instead.

**Comment by cryptiklemur** [comment]
The restcord spec is supposed to follow the official discord docs directly, as its built using their docs

**Comment by cryptiklemur** [comment]
The current createMessage example links directly to the official Discord Create Message reference for body fields, including embeds: https://www.restcord.com/examples/#send-a-message
